tests/test_models.py

```diff
@@ -1084,3 +1137,206 @@ def test_lots_of_sweeps(self):
             random_seed=2,
         )
         assert all(tree.num_roots == 1 for tree in ts.trees())
+
+
+class TestNeutralFixation:
+    """
+    Tests for the neutral fixation model.
+    """
+
+    def test_model_end_broken(self):
+        # Checking that we're correctly detecting the fact that
+        # sweeps are non renentrant.
+        model = msprime.NeutralFixation(
```

Comment:
Totally opaque to me what this test is doing?

Reply:
I think these are copy-pastes of the Sweep selection model tests, and testing a bunch of internal things. This is checking for a known limitation.

Reply:
yep-- testing all this stuff in the `s=0` case. thought these would be good to include, though not necessary.